.vscode/settings.json

```diff
@@ -1,11 +1,17 @@
 {
-    "cSpell.words": [
-        "dotnet",
-        "nuget",
-        "nupkg",
-        "rapid",
-        "rsdl",
-        "tools",
-        "tools rsdl dotnet rapid nupkg"
-    ]
-}
\ No newline at end of file
+  "cSpell.words": [
+    "dotnet",
+    "nuget",
+    "nupkg",
+    "rapid",
+    "rsdl",
+    "tools",
+    "abnf",
+    "eset",
+    "jetsons",
+    "odata",
+    "svel",
+    "xunit"
+  ],
+  "dotnet-test-explorer.testProjectPath": "**/*tests.@(csproj|vbproj|fsproj)"
```

Comment:
Do you want to add cSpell and the dotnet-test-explorer to the list of recommended VS Code extensions in file `.vscode/extensions.json`?

Reply:
Done in latest commit